services/movie.py

```diff
@@ -0,0 +1,33 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    query = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        query = query.filter(genres__id__in=genres_ids,
+                             actors__id__in=actors_ids)
+    elif genres_ids:
+        query = query.filter(genres__id__in=genres_ids)
+    elif actors_ids:
+        query = query.filter(actors__id__in=actors_ids)
```

Comment:
If you delete first condition , nothing will change , cuz your query applies filter on each condition , so you do not need catch first case